frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -581,9 +584,8 @@ def fname(x):
             results1 = sample_to_counts(results1, meas_obj)
             measure_f = qml.counts
 
-        # TODO: dynamic_one_shot_several_mcms is a flaky test.
-        # We remove this test for now and revisit in the future.
-        # validate_measurements(measure_f, shots, results1, results0)
+        validate_measurements(measure_f, shots, results1, results0)
+    '''
```

Comment:
How did this not resolve the issue?

Reply:
I have no idea, but on main latest the flaky failure is still there https://github.com/PennyLaneAI/catalyst/actions/runs/9603583297/job/26488075482

Reply:
Is it actually flaky or is it consistent now?

Reply:
It is flaky, sometimes it succeeds and sometimes it fails

Reply:
Even in the ci jobs, if we just rerun the test there's a good chance it will succeed. I purposefully did not rerun so the error message will stay there